SwiftCompilerSources/Sources/Optimizer/InstructionSimplification/SimplifyAllocStack.swift

```diff
@@ -19,6 +19,7 @@ extension AllocStackInst : Simplifiable, SILCombineSimplifiable {
       return
     }
     _ = optimizeExistential(context)
```

Comment:
If you add `optimizePackElement` you need to return if `optimizeExistential` returns true, because that means that the alloc_stack is already deleted.